app/views/layouts/application.html.erb

```diff
@@ -20,7 +20,7 @@
             <a class="nav-item nav-link " href="">Status</a>
           </div>
           <div class="navbar-nav">
-            <a class="nav-item nav-link " href="">Vendors</a>
+            <%= link_to "Vendors", vendors_url, class: 'nav-item nav-link' %>
```

Comment:
Wire up basic vendor interaction. I found this helpful in testing and it led to me finding bugs.

Reply:
awesome!